server/server.go

```diff
@@ -110,7 +110,8 @@ func (s *Server) HandleEvent(ctx context.Context, e lookout.Event) error {
 	if err == nil {
 		status = models.EventStatusProcessed
 	} else {
-		logger.Errorf(err, "event processing failed")
+
+		ctxlog.Get(ctx).Errorf(err, "event processing failed")
```

Comment:
After extracting the commits to #492, I think this change is not needed anymore

Reply:
that's true